setup.py

```diff
@@ -65,15 +65,15 @@
     "fastapi>=0.68.0",
     "uvicorn[standard]>=0.14.0,<1",
     "gunicorn; platform_system != 'Windows'",
-    "dask[dataframe]>=2021.1.0",
+    "dask[dataframe]>=2024.4.2",
```

Comment:
I feel like I saw an answer in slack somewhere that I can't find at the moment, but this is a pretty big version jump, was Dask having issues with new Python version?

Reply:
We had a issue with Dask with new python version. Build is failing so had to upgrade but i have tried the immediate version so let me give a shot with the version released in 2022.

Reply:
you're good! I was just looking for reason - since we need to upgrade we might as well catch up